web/src/features/panels/search-panel/getSearchData.ts

```diff
@@ -0,0 +1,41 @@
+import { t } from 'i18next';
+import { GridState } from 'types';
+
+import { ZoneRowType } from './ZoneList';
+
+export const getAllZones = () => {
+  // Get all zone data directly from translations
+  const zoneData = t('zoneShortName', { returnObjects: true }) as Record<
+    string,
+    ZoneRowType
+  >;
+
+  return zoneData;
+};
+
+export const getFilteredList = (
+  searchTerm: string,
+  zoneData: Record<string, ZoneRowType>
+): ZoneRowType[] => {
+  if (!searchTerm) {
```

Comment:
We should probably require the length of the term to be above 1 so we don't match every singe zone that has an a in it for example.

It should not be the same as NoResults though.

Reply:
It's a fair suggestion, but given that it would increase complexity here (another state the search and results list can be in and that requires matching UI) and doesn't really seem to be a problem as long as we still have virtualised list, I'll postpone this for now :)